code-challenges.js

```diff
@@ -29,7 +66,18 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
-
+//Create a function that takes in 1 array argument.
+const minMax = (arr) => {
+    //Initialize a local variable called "newArr" and set it to empty so that when we call on this function, it empties newArr.
+    let newArr = []
+    //Find the smallest value in the arr argument using Math.min.apply() and this largest value in the arr argument using Math.max.apply(). This mehod will use .min to find the smallest value in an list of numbers (CANNOT TAKE IN AN ARRAY), .apply is a convenient way to pass in an array and applies the .min method onto the second argument. We do not use the first argument in .apply because there is no given 'this.' object. .min/.max does not use 'this.' so we set it to null. After finding the smallest and largest value, we push those values into newArr in order by pushing the smallest value first then the largest.
+    newArr.push(Math.min.apply(null, arr))
+    newArr.push(Math.max.apply(null, arr))
+    //Return newArr with its 2 values.
+    return newArr
+}
+console.log(minMax(nums1));
+console.log(minMax(nums2));
 
```

Comment:
Interesting approach. Nice explanation of the methods.

I think the spread operator could be a good refactor.
    newArr.push(Math.min(...arr))
    newArr.push(Math.max(...arr))